firefox-ios/firefox-ios-tests/Tests/XCUITests/TrackingProtectionTests.swift

```diff
@@ -214,9 +214,7 @@ class TrackingProtectionTests: BaseTestCase {
         )
         mozWaitForElementToExist(app.staticTexts.elementContainingText("Firefox has not connected to this website."))
 
-        // The lock icon is no longer here.
-        // https://github.com/mozilla-mobile/firefox-ios/issues/22600
-        // XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].label, "Connection not secure")
+        XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].label, "Connection not secure")
```

Comment:
I confirm that this line is now applicable: The lock icon is present for URL with a bad cert.

Reply:
Thank you @clarmso